### DISCUSSION TITLE
Channel estimation for sionna tutorial 4

### DISCUSSION THREAD

**Discussion by chrisDeCstV** [NVlabs/sionna#42] [Show and tell]
![chan_est_sionna](https://user-images.githubusercontent.com/39466346/184572577-3d1fdfd2-e32e-47ee-8fd3-e6f14345192e.JPG)

Hi all, 

Sionna tutorial 4 contains 3 link curves - with perfect channel, with neural receiver and with "LS estimated channel", but in fact there's little estimation going on over there. The results suggest that the neural receiver outperforms a "normal" channel estimation method ( since equalization is trivial for single layer). 

The truth is that even with a half-designed interpolator for chest, the performance of a "classical" receiver and neural ones are very similar to each other. 

Thank you ! 
Chris

**Comment by jhoydis** [comment]
Hi Chris,

The baseline we have chosen in this tutorial example is indeed simple and many improvements are possible.

The goal of this tutorial is to show how to train a neural receiver that can come close to perfect CSI performance; not that there aren't any alternative approaches with similar performance.

Since the tutorial was written, we have added a [linear interpolator](https://nvlabs.github.io/sionna/api/ofdm.html#sionna.ofdm.LinearInterpolator) which might provide slightly improved performance in this scenario.

Can you provide details about the channel estimation algorithm that you have implemented? We are always looking for good baseline algorithms that could be added to Sionna for benchmarking.
